2017-07-20 17:44:42,296 - FileController.py - INFO: 读取yaml文件：../TestCase/Config/config.yaml
2017-07-20 17:44:42,299 - FileController.py - INFO: 创建yaml文件：D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-20 17:44:42,299 - CaseGoto.py - INFO: 临时文件成功： D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-20 17:44:42,299 - CaseGoto.py - INFO: Header 配置文件中未配置，可能引起出错
2017-07-20 17:44:42,306 - FileController.py - INFO: 读取excel文件：D:\GitPro\Python\PythonApiTest\TestCase\TestData\Qone.xlsx
2017-07-20 17:44:42,307 - CaseGoto.py - INFO: CaseList: {'CasePath': 'D:\\GitPro\\Python\\PythonApiTest\\TestCase\\TestData\\Qone.xlsx_case', 'CaseNo': 'No.1', 'API_Purpose': '登录', 'Request_Url': '/api/CobraUser/Login', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '$deviceversion$', 'FORMAT': 'json', 'TELPHONE': '$account$', 'TIMESTAMP': '$tm$', 'VERSION': '$veron$', 'U_ID': 'ANDROID', 'PASSWORD': '$password$', 'Sign': '$sign$'}, 'Need_Collection': 'ERROR_CODE', 'Depends': '', 'Response_Type': 'Json', 'Checkpoint': {'ERROR_MESSAGE': '有新版本，请检查更新！', 'ERROR_CODE': '10006'}, 'Active': 'TRUE', 'Temp_Filepath': 'D:\\GitPro\\Python\\PythonApiTest\\output\\tempyaml.yaml'}
2017-07-20 17:44:42,307 - CaseGoto.py - INFO: CaseList: {'CasePath': 'D:\\GitPro\\Python\\PythonApiTest\\TestCase\\TestData\\Qone.xlsx_case', 'CaseNo': 'No.2', 'API_Purpose': '登录', 'Request_Url': '/api/CobraUser/Login1', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '$deviceversion$', 'FORMAT': 'json', 'TELPHONE': '$account$', 'TIMESTAMP': '$tm$', 'VERSION': '$veron$', 'U_ID': 'ANDROID', 'PASSWORD': '$password$', 'Sign': '$sign$'}, 'Need_Collection': '', 'Depends': '/api/CobraUser/Login', 'Response_Type': 'Json', 'Checkpoint': {'ERROR_MESSAGE': '有新版本，请检查更新！', 'ERROR_CODE': '10006'}, 'Active': 'TRUE', 'Temp_Filepath': 'D:\\GitPro\\Python\\PythonApiTest\\output\\tempyaml.yaml'}
2017-07-20 17:44:42,308 - CaseInteg.py - INFO: {'CasePath': 'D:\\GitPro\\Python\\PythonApiTest\\TestCase\\TestData\\Qone.xlsx_case', 'CaseNo': 'No.1', 'API_Purpose': '登录', 'Request_Url': '/api/CobraUser/Login', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '$deviceversion$', 'FORMAT': 'json', 'TELPHONE': '$account$', 'TIMESTAMP': '$tm$', 'VERSION': '$veron$', 'U_ID': 'ANDROID', 'PASSWORD': '$password$', 'Sign': '$sign$'}, 'Need_Collection': 'ERROR_CODE', 'Depends': '', 'Response_Type': 'Json', 'Checkpoint': {'ERROR_MESSAGE': '有新版本，请检查更新！', 'ERROR_CODE': '10006'}, 'Active': 'TRUE', 'Temp_Filepath': 'D:\\GitPro\\Python\\PythonApiTest\\output\\tempyaml.yaml'}
2017-07-20 17:44:42,308 - DataGenerate.py - ERROR: 获取参数值失败,可能含有二次包装数据,请再次调用 {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '$sign$'}
2017-07-20 17:44:42,308 - CaseInteg.py - INFO: 请求表单已处理
2017-07-20 17:44:42,308 - CaseInteg.py - INFO: {'Request_Url': '/api/CobraUser/Login', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '$sign$'}}
2017-07-20 17:44:42,309 - UserParam.py - INFO: B4B83DC748B8E511A2846C0B840997C3APP_KEY10001DEVICE_VERSION107FORMATjsonPASSWORDqaz123TELPHONE13817645775TIMESTAMP2017-07-20 17:44:42U_IDANDROIDVERSION1.0B4B83DC748B8E511A2846C0B840997C3
2017-07-20 17:44:42,309 - DataGenerate.py - INFO: {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '00BF7D323163E8071B1A055271205FC3'}
2017-07-20 17:44:42,309 - RequestGenerate.py - INFO: {'url': 'https://sitqone-wh.quarkfinance.com/api/CobraUser/Login', 'method': 'post', 'headers': '', 'json': {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '00BF7D323163E8071B1A055271205FC3'}}
2017-07-20 17:44:42,309 - CaseInteg.py - INFO: 请求数据已解析
2017-07-20 17:44:42,675 - RequestGenerate.py - INFO: <Response [200]>
2017-07-20 17:44:42,675 - CaseInteg.py - INFO: 请求数据已发送
2017-07-20 17:44:42,675 - CaseInteg.py - INFO: 请求返回数据：<Response [200]>
2017-07-20 17:44:42,675 - test_ApiTest_2.py - INFO: 返回值Json字符串：{'BODY': '{"RESULT":true,"MESSAGE":"有新版本需要更新","APP_DOWNLOAD_URL_ANDROID":"https://sitqone-wh-resources.quarkfinance.com/resources/packages/qone.apk"}', 'RESPONSE_STATUS': 'BadRequest', 'ELAPSE_TIME': None, 'ERROR_CODE': '10006', 'ERROR_MESSAGE': '有新版本，请检查更新！'}
2017-07-20 17:44:42,675 - test_ApiTest_2.py - INFO: {'$ERROR_CODE$': '10006'}
2017-07-20 17:44:42,675 - ResultGenerate.py - INFO: CasePath:D:\GitPro\Python\PythonApiTest\TestCase\TestData\Qone.xlsx_case	CaseNumb:No.1	CaseName:登录	Request_Url:/api/CobraUser/Login	Temp_Filepath:D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml	caseresult:Pass	
2017-07-20 17:44:42,676 - FileController.py - INFO: 写入yaml文件：D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-20 17:44:42,677 - test_ApiTest_2.py - INFO: 返回值已处理
2017-07-20 17:44:42,678 - FileController.py - INFO: 读取yaml文件：D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-20 17:44:42,679 - CaseInteg.py - INFO: {'CasePath': 'D:\\GitPro\\Python\\PythonApiTest\\TestCase\\TestData\\Qone.xlsx_case', 'CaseNo': 'No.2', 'API_Purpose': '登录', 'Request_Url': '/api/CobraUser/Login1', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '$deviceversion$', 'FORMAT': 'json', 'TELPHONE': '$account$', 'TIMESTAMP': '$tm$', 'VERSION': '$veron$', 'U_ID': 'ANDROID', 'PASSWORD': '$password$', 'Sign': '$sign$'}, 'Need_Collection': '', 'Depends': '/api/CobraUser/Login', 'Response_Type': 'Json', 'Checkpoint': {'ERROR_MESSAGE': '有新版本，请检查更新！', 'ERROR_CODE': '10006'}, 'Active': 'TRUE', 'Temp_Filepath': 'D:\\GitPro\\Python\\PythonApiTest\\output\\tempyaml.yaml'}
2017-07-20 17:44:42,679 - DataGenerate.py - ERROR: 获取参数值失败,可能含有二次包装数据,请再次调用 {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '$sign$'}
2017-07-20 17:44:42,679 - CaseInteg.py - INFO: 请求表单已处理
2017-07-20 17:44:42,679 - CaseInteg.py - INFO: {'Request_Url': '/api/CobraUser/Login1', 'Request_method': 'POST', 'Header': '', 'Body_Type': 'Json', 'Request_Body': {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '$sign$'}}
2017-07-20 17:44:42,679 - UserParam.py - INFO: B4B83DC748B8E511A2846C0B840997C3APP_KEY10001DEVICE_VERSION107FORMATjsonPASSWORDqaz123TELPHONE13817645775TIMESTAMP2017-07-20 17:44:42U_IDANDROIDVERSION1.0B4B83DC748B8E511A2846C0B840997C3
2017-07-20 17:44:42,679 - DataGenerate.py - INFO: {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '00BF7D323163E8071B1A055271205FC3'}
2017-07-20 17:44:42,680 - RequestGenerate.py - INFO: {'url': 'https://sitqone-wh.quarkfinance.com/api/CobraUser/Login1', 'method': 'post', 'headers': '', 'json': {'APP_KEY': '10001', 'DEVICE_VERSION': '107', 'FORMAT': 'json', 'TELPHONE': '13817645775', 'TIMESTAMP': '2017-07-20 17:44:42', 'VERSION': '1.0', 'U_ID': 'ANDROID', 'PASSWORD': 'qaz123', 'Sign': '00BF7D323163E8071B1A055271205FC3'}}
2017-07-20 17:44:42,680 - CaseInteg.py - INFO: 请求数据已解析
2017-07-20 17:44:42,714 - RequestGenerate.py - INFO: <Response [404]>
2017-07-20 17:44:42,714 - CaseInteg.py - INFO: 请求数据已发送
2017-07-20 17:44:42,714 - CaseInteg.py - INFO: 请求返回数据：<Response [404]>
2017-07-20 17:44:42,714 - test_ApiTest_2.py - INFO: 用例文件中未配置需要收集的参数，若有后续依赖，执行会报错
2017-07-20 17:44:42,714 - test_ApiTest_2.py - INFO: 返回值Json字符串：{'Message': '找不到与请求 URI“http://sitqone-wh.quarkfinance.com/api/CobraUser/Login1”匹配的 HTTP 资源。'}
2017-07-20 17:44:42,714 - ResultGenerate.py - INFO: CasePath:D:\GitPro\Python\PythonApiTest\TestCase\TestData\Qone.xlsx_case	CaseNumb:No.2	CaseName:登录	Request_Url:/api/CobraUser/Login1	Temp_Filepath:D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml	caseresult:Fail	
2017-07-20 17:44:42,714 - FileController.py - INFO: 写入yaml文件：D:\GitPro\Python\PythonApiTest\output\tempyaml.yaml
2017-07-20 17:44:42,716 - test_ApiTest_2.py - INFO: 返回值已处理
